### ISSUE TITLE
NameError: undefined local variable or method `state' for ActiveSupport::IsolatedExecutionState:Module

### ISSUE THREAD

**Issue by sebitoelcheater** [rails/rails#44407]
### Steps to reproduce

Update to rails 7.0.2.1

enqueue an ActiveJob

### Expected behavior

Execute the job

### Actual behavior

```
NameError: undefined local variable or method `state' for ActiveSupport::IsolatedExecutionState:Module
  activesupport (7.0.2.1) lib/active_support/isolated_execution_state.rb:41:in `key?'
  activesupport (7.0.2.1) lib/active_support/execution_wrapper.rb:120:in `active?'
  activesupport (7.0.2.1) lib/active_support/execution_wrapper.rb:88:in `wrap'
  activesupport (7.0.2.1) lib/active_support/reloader.rb:71:in `wrap'
  sidekiq (6.4.1) lib/sidekiq/rails.rb:13:in `call'
...
(14 additional frame(s) were not displayed)
```

### System configuration
**Rails version**: 7.0.2.1

**Ruby version**: 3.0.2

**Comment by SkipKayhil** [comment]
This should be fixed in 7.0.2.2 https://my.diffend.io/gems/activesupport/7.0.2.1/7.0.2.2

**Comment by byroot** [comment]
Yes, the fix in `7.0.2.1` had a problem, `7.0.2.2` went out just a few minutes later.
